core/src/main/java/io/paradoxical/cassieq/dataAccess/AccountRepositoryImpl.java

```diff
@@ -0,0 +1,69 @@
+package io.paradoxical.cassieq.dataAccess;
+
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.querybuilder.Insert;
+import com.datastax.driver.core.querybuilder.QueryBuilder;
+import com.datastax.driver.core.querybuilder.Select;
+import com.google.common.collect.ImmutableSet;
+import com.google.inject.Inject;
+import io.paradoxical.cassieq.dataAccess.interfaces.AccountRepository;
+import io.paradoxical.cassieq.model.accounts.AccountDefinition;
+import io.paradoxical.cassieq.model.accounts.AccountKey;
+import io.paradoxical.cassieq.model.accounts.AccountName;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.*;
+import static java.util.stream.Collectors.toList;
+
+import java.security.SecureRandom;
+import java.util.List;
+import java.util.Optional;
+
+public class AccountRepositoryImpl extends RepositoryBase implements AccountRepository {
+
+    private final Session session;
+    private static final SecureRandom secureRandom = new SecureRandom();
+
+    @Inject
+    public AccountRepositoryImpl(final Session session) {
+        this.session = session;
+    }
+
+    @Override
+    public Optional<AccountDefinition> createAccount(final AccountName accountName) {
+        AccountKey primaryKey = AccountKey.random(secureRandom);
```

Comment:
it might be worth putting in some comments about what the primarykey/secondary key are, cause i have no idea